doto-frontend/src/components/ModalContent.js

```diff
@@ -0,0 +1,138 @@
+                                    root: classes.taskNameInput,
+                                    shrink: classes.labelFocus,
+                                },
+                            }}
+                        />
+                    </div>
+                    <div>
+                        <TextField className="text-area group-spacing" id="standard-basic" label="Task description" />
+                    </div>
+                    <div className="spacing">
+                        <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                            <KeyboardDatePicker
+                                disableToolbar
+                                variant="inline"
+                                format="MM/dd/yyyy"
+                                margin="normal"
+                                id="date-picker-inline"
+                                label="Due Date"
+                                value={selectedDate}
+                                onChange={handleDateChange}
+                                KeyboardButtonProps={{
+                                    "aria-label": "change date",
+                                }}
+                            />
+                        </MuiPickersUtilsProvider>
+                    </div>
+                    <div>
+                        <TextField className="small-text-area group-spacing" id="standard-basic" label="Duration" />
+                    </div>
+                    <div>
+                        <TextField className="text-area spacing" id="standard-basic" label="Location" />
+                    </div>
+                    <div className="drop-down">
+                        <FormControl className={classes.formControl}>
+                            <InputLabel id="priority-label">Priority</InputLabel>
+                            <Select value={priority} onChange={handlePriority}>
+                                <MenuItem value={10}>High</MenuItem>
+                                <MenuItem value={20}>Medium</MenuItem>
+                                <MenuItem value={30}>Low</MenuItem>
+                            </Select>
+                        </FormControl>
+                    </div>
+                    <div className="drop-down">
+                        <FormControl className={classes.formControl}>
+                            <InputLabel id="reminder-label">Reminders</InputLabel>
+                            <Select value={reminder} onChange={handleReminder}>
+                                <MenuItem value={10}>1 Week Before</MenuItem>
+                                <MenuItem value={20}>1 Day Before</MenuItem>
+                                <MenuItem value={30}>1 Hour Before</MenuItem>
+                                <MenuItem value={40}>30 Minuties Before</MenuItem>
+                                <MenuItem value={50}>15 Minuties Before</MenuItem>
+                                <MenuItem value={60}>5 Minuties Before</MenuItem>
```

Comment:
Minutes is spelt incorrectly for all the 3 options

Reply:
Good catch I'll fix this in a sec 👍